tests/molecules/C4_ae/SD-C4_AE_Mol_QP.wfnoj.xml

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0"?>
 <qmcsystem>
   <wavefunction name="psi0" target="e">
-    <determinantset type="MolecularOrbital" name="LCAOBSet" source="ion0" transform="yes" cuspCorrection="yes" href="C4_AE_Mol_QP.h5">
+    <sposet_collection type="MolecularOrbital" name="LCAOBSet" source="ion0" transform="yes" cuspCorrection="yes" href="C4_AE_Mol_QP.h5">
+      <sposet name="spo-up" size="92" cuspInfo="spo-up.cuspInfo.xml">
```

Comment:
Can we keep this 12 as it was and also keep using the pre-computed cuspInfo xml file?

Reply:
I'll revert this change and add a new test